pkg/agent/openflow/client.go

```diff
@@ -1234,6 +1247,22 @@ func (c *client) SendTraceflowPacket(dataplaneTag uint8, packet *binding.Packet,
 	return c.bridge.SendPacketOut(packetOutObj)
 }
 
+func (c *client) InstallPacketCaptureFlows(dataplaneTag uint8, senderOnly, receiverOnly bool, packet *binding.Packet, endpointPackets []binding.Packet, ofPort uint32, timeoutSeconds uint16) error {
+	cacheKey := fmt.Sprintf("%x", dataplaneTag)
+	var flows []binding.Flow
+	for _, f := range c.packetCaptureFeatures {
+		flows = append(flows, f.flowsToCapture(dataplaneTag,
+			c.ovsMetersAreSupported,
+			senderOnly,
+			receiverOnly,
+			packet,
+			endpointPackets,
+			ofPort,
+			timeoutSeconds)...)
```

Comment:
I don't quite understand this handling. Unlike Traceflow which needs to report how packet is processed in each stage, why we need to install flows for different features for packet capture? Will users get a single packet for each packet or multiple duplicate ones reported by each feature? If the former, why we need such method?

Reply:
updated.
https://github.com/antrea-io/antrea/pull/5821/commits/223bd5cc3703e2cd8872e2089f8b4111399a13df